_commands/basics/date.md

```diff
@@ -3,22 +3,32 @@
 
 date
 -------
-TODO: Add documentation for this command by submitting a pull request.
+The date command is used to print out, or change the value of, the system's time and date information
 <!-- one line explanation would go here -->
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$date
+output - Thu Feb 8 16:47:32 MST 2001
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
+Format -- a sequence of characters which specifies how output will appear:
+"%%" - A literal percent sign ("%")
+%a - The abbreviated weekday name (Mon, Tues, Wed, Th, Fri)
+%A - The full weekday name (Sunday, Monday, Tuesday, Wednesday, Thursday, Friday)
+%b - The abbreviated month name 
+%B - Locale's full month name
+%H - Hour(00..23)
+%n - A newline
+%T - Time; same as %H:%M:%S
+%Y - year
 
 #### Example command
-
-##### Break it down
-
-#### Example command
-
+ls -al > output_$(date+"%m_%d_%Y")
 ##### Break it down
```

Comment:
You'll need to insert a new line here, between the example command and the `#### Break it down`, because the latter isn't rendering as a section title properly
![image](https://user-images.githubusercontent.com/11037821/31828354-fd1f99b4-b587-11e7-8d38-9fba856735f3.png)